hw/analog/adc.sp

```diff
@@ -0,0 +1,94 @@
+** Copyright EPFL contributors.
```

Comment:
please move the analog folder under ip_examples/ams , so it should be

`hw/ip_examples/ams/analog`